python/python/ert_gui/simulation/models/iterated_ensemble_smoother.py

```diff
@@ -76,32 +76,35 @@ def runSimulations(self, arguments):
         analysis_iter_config = analysis_config.getAnalysisIterConfig()
         num_retries_per_iteration = analysis_iter_config.getNumRetries()
         num_tries = 0
-        current_iteration = 1
 
-        while current_iteration <= getNumberOfIterations() and num_tries < num_retries_per_iteration:
+        while True:
             pre_analysis_iter_num = analysis_module.getInt("ITER")
             self.analyzeStep( run_context )
-            post_analysis_iter_num = analysis_module.getInt("ITER")
+            current_iter = analysis_module.getInt("ITER")
 
             analysis_success = False
-            if  post_analysis_iter_num > pre_analysis_iter_num:
+            if current_iter > pre_analysis_iter_num:
```

Comment:
Could not these lines be replaced by;
 ```python
analysis_success = current_iter > pre_analysis_iter_num
if analysis_success:
    .....
```
?

Reply:
Yes - done.